lib/web/auth/saml/index.js

```diff
@@ -17,6 +17,7 @@ passport.use(new SamlStrategy({
   entryPoint: config.saml.idpSsoUrl,
   issuer: config.saml.issuer || config.serverURL,
   cert: fs.readFileSync(config.saml.idpCert, 'utf-8'),
+  privateCert: config.saml.clientCert === undefined ? undefined : fs.readFileSync(config.saml.clientCert, 'utf-8'),
```

Comment:
I am not sure but shouldn't this default to default cert bundle if undefined?

Reply:
But then you would need another config-option to disable signing.

Reply:
This should NOT(!) fallback to the the default cert bundle. That's not how SAML works.

Reply:
We should verify that the file exists before loading it and provide a proper error instead of crashing with a stacktrace.

Reply:
Now there is error handling with logging...